src/compiler/crystal/command.cr

```diff
@@ -525,9 +519,39 @@ class Crystal::Command
     values
   end
 
+  private def color_option(opts)
+    opts.on("--color [auto|always|never]", "Enable colored output") do |color|
+      color = "always" if color.empty?
+      case color
+      when "auto"
+        Colorize.on_tty_only!
+      when "always"
```

Comment:
`color = "always" if color.empty?` => `when "always", .empty?`

Reply:
`when "always", ""`

Reply:
Personally, `color = "always" if color.empty?` shows `"always"` as default clearly, so I would keep this.

Reply:
Maybe, but it adds an completely unnecessary comparison and assignment.

Reply:
It should be all in one `case` statement, not a `case` plus an `if`.